File: src/main/java/io/renren/modules/oss/entity/SysOssEntity.java
package io.renren.modules.oss.entity;

import lombok.Data;

import java.io.Serializable;
import java.util.Date;


/**
 * 文件上传
 *
 * @author ChunLiang Hu
 * @email davichi2009@gmail.com
 * @date 2017-03-25 12:13:26
 */
@Data
public class SysOssEntity implements Serializable {
    private static final long serialVersionUID = 1L;

    /**
     * 主键ID
     */
    private Long id;
    /**
     * URL地址
     */
    private String url;
    /**
     * 创建时间
     */
    private Date createDate;

}

File: src/main/java/io/renren/modules/oss/entity/SysOssEntity.kt
package io.renren.modules.oss.entity

import java.io.Serializable
import java.util.*


/**
 * 文件上传
 *
 * @author ChunLiang Hu
 * @email davichi2009@gmail.com
 * @date 2017-03-25 12:13:26
 */
data class SysOssEntity(
        private val serialVersionUID: Long = 1L,
        /**
         * 主键ID
         */
        var id: Long? = null,

        /**
         * URL地址
         */
        var url: String? = null,

        /**
         * 创建时间
         */
        var createDate: Date? = null

) : Serializable
